Specs/DateTime/Dutch/DateTimeModel.json

```diff
@@ -702,6 +686,16 @@
               "timex": "XXXX-10-14T08:00:31",
               "type": "datetime",
               "value": "2017-10-14 08:00:31"
+            },
+            {
+              "timex": "XXXX-10-14T20:00:31",
```

Comment:
```"Input": "Ik ga 14 oktober om 8:00:31 terug"```

@tellarin what is the resolution for other (European) locales? I think this enhancement should be logged in a feature request. In locales where they use 24 hour notation, 8:00:31 should maybe not be mapped to 20:00:31. However, '8 uur' (spoken language) should be mapped to both.

Not an issue for this PR, but maybe good to already log it as low prio case? Would be good to see feedback eventually.

Reply:
Yes, we should open an issue for this as low-priority.